constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala

```diff
@@ -17,36 +17,50 @@
 package com.tecsisa.constructr.coordination
 package consul
 
+import java.nio.charset.StandardCharsets.UTF_8
+import java.util.Base64.{ getUrlDecoder, getUrlEncoder }
+
 import akka.Done
-import akka.actor.ActorSystem
+import akka.actor.{ ActorSystem, Address, AddressFromURIString }
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.client.RequestBuilding.{ Get, Put }
 import akka.http.scaladsl.model.MediaTypes.`application/json`
 import akka.http.scaladsl.model.StatusCodes.{ NotFound, OK }
-import akka.http.scaladsl.model.{ HttpEntity, HttpRequest, HttpResponse, RequestEntity, ResponseEntity, StatusCode, Uri }
+import akka.http.scaladsl.model.{
```

Comment:
Please, review this formatting. The indentation is ugly. Use a couple of lines if necessary.